src/sedcli_main.c

```diff
@@ -349,6 +395,103 @@ static void print_sed_status(int status)
 	}
 }
 
+static void sed_discv_print_normal(struct sed_opal_level0_discovery *discv, const char *dev_path)
+{
+	bool locking_enabled;
+	uint16_t comid;
+
+	locking_enabled = discv->sed_locking.locking_en ? true : false;
+	comid = discv->sed_opalv200.base_comid;
+
+	if (!comid)
+		sedcli_printf(LOG_INFO, "%s: is NOT SED-OPAL Compliant.\n", dev_path);
+	else
+		sedcli_printf(LOG_INFO, "%s: is SED-OPAL Compliant.\n", dev_path);
+
+	if (locking_enabled)
+		sedcli_printf(LOG_INFO, "%s: is provisioned with Locking Enabled\n", dev_path);
+	else
+		sedcli_printf(LOG_INFO, "%s: is NOT provisioned with Locking Unabled\n", dev_path);
+}
+
+#define SED_ENABLE "ENABLED"
+#define SED_DISABLE "DISABLED"
+
+char *DEV_SED_COMPATIBLE;
+char *DEV_SED_LOCKED;
+
+static void sed_discv_print_udev(struct sed_opal_level0_discovery *discv)
+{
+	bool locking_enabled;
+	uint16_t comid;
+
+	locking_enabled = discv->sed_locking.locking_en ? true : false;
+	comid = discv->sed_opalv200.base_comid;
+
+	if (!comid)
+		DEV_SED_COMPATIBLE = SED_DISABLE;
+	else
+		DEV_SED_COMPATIBLE = SED_ENABLE;
+
+	if (locking_enabled)
+		DEV_SED_LOCKED = SED_ENABLE;
+	else
+		DEV_SED_LOCKED = SED_DISABLE;
+
+	sedcli_printf(LOG_INFO, "DEV_SED_COMPATIBLE=%s\n", DEV_SED_COMPATIBLE);
+	sedcli_printf(LOG_INFO, "DEV_SED_LOCKED=%s\n", DEV_SED_LOCKED);
+}
+
+static int handle_sed_discv(void)
+{
+	int ret = 0;
+
+	#ifdef CONFIG_OPAL_DRIVER
```

Comment:
I would prefer that preprocessor macro to be in the array that defines actual command. In case of usage of OPAL_DRIVER simply there wouldn't be discovery command